plopfile.js

```diff
@@ -54,12 +54,23 @@ module.exports = (plop) => {
         if (fs.existsSync(indexFile)) {
           // Split the index file into lines.
           const lines = fs.readFileSync(indexFile, 'utf8').split('\n');
+          // Only sort lines that begin with "export {" in order to ignore
+          // comments and exported types.
+          const sortableLines = lines.filter((line) =>
+            /^export {.*/g.test(line),
```

Comment:
The only change I made to the logic is now it only sorts lines that start with `export {` (as opposed to just `export`) to avoid moving the type import lines around.